jerry-debugger/src/lib/cdt-proxy.ts

```diff
@@ -92,12 +99,18 @@ export class ChromeDevToolsProxyServer {
     this.api.Debugger.expose({
       enable: notImplemented,
       setBlackboxPatterns: notImplemented,
-      setAsyncCallStackDepth: async (params) => {
-        this.asyncCallStackDepth = params.maxDepth;
-      },
+      stepOver: async () => this.delegate.requestStepOver(),
```

Comment:
I wonder why these callbacks are `async` / need to return a `Promise`.
When are they expected to be resolved? When the action has been performed successfully?
Not sure if it matters.

Reply:
It complains if I don't make them async, but I haven't yet understood why.

Reply:
Interesting question... maybe it's not enough to fire off the request but I might need to wait to find out if it's succeeded or not? I suppose we'll realize that later if it's the case so I'll probably not try to address it right now. Like most software, it's not well documented enough to know:

Here are the docs: :)

**Debugger.stepOver**
Steps over the statement.

Reply:
> It complains if I don't make them async, but I haven't yet understood why.

Probably because that's the type of that property.